### PR TITLE
edit Media Type Object

### PR DESCRIPTION
**Pull request by krishahn**

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -1268,15 +1268,15 @@ content:
...
+<a name="mediaTypeExample"></a>example | Any | Example of the media type.  The example object SHOULD be in the correct format as specified by the media type.  The `example` object is mutually exclusive of the `examples` object.  Furthermore, if referencing a `schema` which contains an example, the `example` value SHALL _override_ the example provided by the schema.
+<a name="mediaTypeExamples"></a>examples | Map[ `string`, [Example Object](#exampleObject) \| [Reference Object](#referenceObject)] | Examples of the media type.  Each example object SHOULD  match the media type and specified schema if present.  The `examples` object is mutually exclusive of the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value SHALL _override_ the example provided by the schema.
+<a name="mediaTypeEncoding"></a>encoding | Map[`string`, [Encoding Object](#encodingObject)] | A map between a property name and its encoding information. The key, being the property name, MUST exist in the schema as a property. The encoding object SHOULD apply only to `requestBody` objects when the content type is `multipart`.
```
Comment (MikeRalphson) [inline versions/3.0.md]: Not splitting infinitives may may make the text easier to read for those whose first language is not English. So prefer 'only apply to' over 'apply only to'.

Comment (krishahn) [inline versions/3.0.md]: tks, i'll stop nit-picking. I thought the adverb had to appear between the preposition and a verb to split it. Regarding the placement of only, this isn't the first time somebody complained about my moving "only" as far to the end of the sentence as possible. I think doing so makes things sound a little British--not good. On the plus side, doing so often prevents embarrassing wording mistakes.

Comment (MikeRalphson) [inline versions/3.0.md]: Disclosure: British. :)

Comment (krishahn) [inline versions/3.0.md]: oops

**Pull request merged**